src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -33,28 +33,22 @@
  * Handles communication with the keyserver.
  */
 public class Rscc {
+  public static final int PACKAGE_SIZE = 10000; // needed, since a static method access it.
```

Comment:
@l-marchesi What about that:
in the model:
`  public static int getPackageSize(){
    return PACKAGE_SIZE;
  }`
or make it a IntegerProperty that you return in this method, wouldn't that be the best for he GUI-implementation?

in the other class:
` public final static int MAX_SEGMENT_SIZE = Rscc.getPackageSize();`

It seens to work, at least for the IDE